### ISSUE TITLE
Blog: laying out the possibilities

### ISSUE THREAD

**Issue by benmiller314** [pitt-cdm/miller2019spring#10]
As you now know, throughout the semester, I will periodically ask you to develop proposals for a collaborative final project that will allow you to integrate – and extend – what you’ve learned about composing digital media. This is the second of those proposals; you can directly reference an idea from [the last one](https://github.com/pitt-cdm/miller2019spring/issues/5), or start fresh. The question is essentially the same:

**What games or branching narratives come to mind as you view to the class's visual arguments?** (The default option for the collaborative unit is to work with [Twine](http://twinery.org/).) Or, to come at it from a different angle, **what further possibilities of graphic design might you want to explore in connection with other people, and other media?**

As always on the forum, replying directly to each other is optional but encouraged: just use the @ symbol to flag someone by username.

**Comment by abrous** [comment]
To be completely honest, I have no idea what projects I may want to do for the final. Since my skills with these programs we are using are limited, I do not know the full extent of what we could get into or use. When a computer type game is mentioned my mind immediately goes to something similar to the Oregon trail game. I am aware it is not too creative but as of right now it is the best option I have. I could create a game about any daily life or maybe incorporate sports in one way or another.

**Comment by jennyaimei120** [comment]
I didn't really have any ideas of what to do for the final project. But when I thought about it for a little bit, I thought what would be cool would be having some type of game that would let you choose what your outcome would be. Not just a “day in your life”  we could do something like pick what you want to eat, what you should chose to do with your friends that day, or even pick random things to pick where you are going on vacation.

**Comment by bcy5** [comment]
I think that one cool idea that I have in mind in terms of working on the final project is that we can incorporate sound and visual rhetoric components into our final video game project. We could have the character make choices based on the sound that they hear or certain visual ques that they see. I still have very little planned for the final project, but I think that's the direction I want to go toward!

**Comment by dashn98** [comment]
For the final project I think incorporating sound will be very important. I agree that a game could incorporate sound and visuals very well. When you are playing a game there is usually some sort of sounds that represent what is actually happening. I think this could be done in a lot of different types of games. For example, even in a simple bowling game we would have visuals of a bowling alley, sounds of pins being knocked over etc.  I'm open to a lot of different game story lines.

**Comment by ghost** [comment]
I'm not totally sure what I'd want to do as a group project since my skills with these applications are really limited. I still really like the "Bandersnatch" idea where you could choose the outcome of the game you're playing. It could be a game where you're working towards a goal or it could be something simple like "a day in the life" where you're just picking things like what you eat for breakfast and whether or not you decide to wear a hat outside.

**Comment by esc37** [comment]
I still think the "choose your own" storyboard idea for the group project is a good idea, kind of like the game of Life. This could incorporate both audio and visual components. However, I'm not sure how difficult creating the visual components of this would be, as we would need to include many options for the game, and include many branching paths.

**Comment by christyh310** [comment]
for the final project, perhaps we could let the user/player select different elements to be combined in a final composition - like background music, or even the title of our soundscapes, and then certain images, and then we could have 5-10 resulting compositions that come up. this would be more of a demo for future students of the class or future designers to be like " this is what could happen when composing digital media", for them to see the possibilities. However, if we truly want variety but also quality, this would be difficult to pull off, because that means several different compositions/arguments would have to be created.

**Comment by eaj34** [comment]
For the final project, I wouldn't mind using sound again, however, I am not entirely a fan of GIMP. I was able to use Audacity fairly easy, but I'm finding myself struggling with GIMP. If the images were simple, I would be able to do them! As of now, if I had to create a project, I would do a kind of game where an image is placed in front of you and you have to pick the item of the sound you hear. People may be surprised when they find out one sound belongs to an item they would suspect. Overall, I'm still not sure on what I would like to do for the project until I get closer and more skilled in the programs.

**Comment by christyh310** [comment]
@eaj34 I totally agree with this - audacity was much easier to use and apply than using/applying different techniques in GIMP
